app/forms/incomeSources/cease/IncomeSourceEndDateForm.scala

```diff
@@ -30,9 +30,11 @@ import services.DateService
 import java.time.LocalDate
 import javax.inject.Inject
 
-class IncomeSourceEndDateForm @Inject()(val dateService: DateService)(implicit val appConfig: FrontendAppConfig) extends CustomConstraints with FeatureSwitching{
+class IncomeSourceEndDateForm @Inject()(val dateService: DateService)(implicit val appConfig: FrontendAppConfig) extends CustomConstraints with FeatureSwitching {
 
-  val dateMustBeComplete = "dateForm.error.dayMonthAndYear.required"
+  val dateMustBeCompleteSE = "dateForm.error.dayMonthAndYear.required.se"
```

Comment:
This change is in the incomeSources (old journey) domain. So here the old journey is being updated with the new journey content. If we do this we might get more entangled with the old and new journeys. I think it would be better if dateForm.error.dayMonthAndYear.required was kept as is, but the deleted message in the messages/welsh messages file was re-added?

Reply:
Yep, no specific text. It's just that the new journey was a replica of the old journey, so the new version is the updated one, and this file will be deleted soon, so while it's not wrong, there's not a massive benefit to implementing the new logic. I don't know if design would want parts of the new design in the old (currently in prod) version or not (for consistency purposes maybe)